.github/workflows/php.yml

```diff
@@ -0,0 +1,38 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Setup PHP with pre-release PECL extension
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '7.4'
```

Comment:
Should there be more supported versions here ? I don't know the exact specs for this lib, but I think this Lib supports a bunch of PHP Versions.

Reply:
I've added PHP 7.0, 7.1, 7.2, 7.3 and 7.4. These match the version constraints of PHPUnit v6.x and php-cs-fixer v2.16.

Reply:
Should I add a cron check as well?

Reply:
I'mma do it. It'll make a it a little bit safer